bootstrap/pkg/kfapp/apply/apply.go

```diff
@@ -0,0 +1,103 @@
+package apply
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	kftypes "github.com/kubeflow/kubeflow/bootstrap/v3/pkg/apis/apps"
+	kfdefsv3 "github.com/kubeflow/kubeflow/bootstrap/v3/pkg/apis/apps/kfdef/v1alpha1"
+	"github.com/kubeflow/kubeflow/bootstrap/v3/pkg/kfapp/coordinator"
+	log "github.com/sirupsen/logrus"
+)
+
+// BootstrapKubeflow is used by the kfctl apply sub-command to take in a configfile
+// as a flag and boostrap a KfApp and deploy it
+func BootstrapKubeflow(configFilePath string, kfResource kftypes.ResourceEnum) error {
+	// Set default app name to kf-app
+	appName := "kf-app"
+
+	// Construct KfDef from the configFilePath provided
+	kfDef := &kfdefsv3.KfDef{}
+	kfDef, err := kfdefsv3.LoadKFDefFromURI(configFilePath)
+	if err != nil {
+		log.Printf("Unable to create KfDef from config file: %v", err)
+	}
+	if kfDef.Name != "" {
+		log.Warnf("Overriding KfDef.Spec.Name; old value %v; new value %v", kfDef.Name, appName)
+	}
+	kfDef.Name = appName
+	isValid, msg := kfDef.IsValid()
+	if !isValid {
+		log.Printf("Invalid kfdef: %v", isValid)
+		log.Printf("Error validating generated KfDef, please check config file validity: %v", msg)
+	}
+	cwd, err := os.Getwd()
+	if err != nil {
+		log.Printf("Error getting current working directory: %v", err)
+	}
+	fmt.Println("Present working directory is: %v", cwd)
```

Comment:
Can you use log instead of fmt and change all instances of Println to a log-level function? (eg Info, Warn).